dataGenerator/src/main/java/com/filip/datagenerator/KafkaListeners.java

```diff
@@ -0,0 +1,49 @@
+package com.filip.datagenerator;
+
+import com.filip.datagenerator.Service.DataGenerator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class KafkaListeners {
+
+    private static Thread offlineTxWork;
+    private static boolean working = false;
+
+    @Autowired
+    DataGenerator dataGenerator;
+
+    @KafkaListener(
+            topics = "bank_working",
+            groupId = "groupId",
```

Comment:
Make this all configurable

Reply:
Tried setting groupId in @KafkaListener to a configured value via @Value -> 
error: `Attribute value must be constant` 

Im guessing @KafkaListener does not accept configurable values.

![image](https://user-images.githubusercontent.com/64900037/168259092-44744217-abf5-4d36-a0bb-dbe6d3ce0680.png)